src/main/java/com/vinylteam/vinyl/PropertiesReader.java

```diff
@@ -87,4 +90,11 @@ public String getAppPort() {
         }
     }
 
+    private void validateInputStream(InputStream inputStream) {
+        if (inputStream == null) {
+            RuntimeException e = new RuntimeException();
+            logger.error(".properties file not found.", e);
+            throw e;
```

Comment:
throw new RuntimeException(); - i think we should write this code after logger.error. Maybe it will be better this way?